src/library_webgl.js

```diff
@@ -4207,7 +4206,7 @@ function createGLPassthroughFunctions(lib, funcs) {
       let cName = name;
       if (name.includes('[')) {
         cName = name.replace('[', '').replace(']', '');
-        name = cName.substr(0, cName.length-1);
+        name = cName.slice(0, -1)
```

Comment:
Removing semicolons at the end seems nonidiomatic JS, here and above?

Reply:
Oops, didn't mean to do that.